lxc/config_trust.go

```diff
@@ -526,20 +526,23 @@ func (c *cmdConfigTrustRemove) Run(cmd *cobra.Command, args []string) error {
 	}
 
 	// Parse remote
-	remote := ""
-	if len(args) > 0 {
-		remote = args[0]
-	}
-
-	resources, err := c.global.ParseServers(remote)
+	resources, err := c.global.ParseServers(args[0])
 	if err != nil {
 		return err
 	}
 
 	resource := resources[0]
 
+	// Support passing <remote>:<fingerprint>
```

Comment:
Wouldn't using `resource.name` be more proper here, rather than repeating the parsing that `ParseServers` has done above?

E.g. like https://github.com/lxc/lxd/blob/8e73e8d0c8d992da2306080c73a3179128965352/lxc/cluster_group.go#L284-L286

Reply:
We need to support:
```
lxc config trust remove REMOTE: FINGERPRINT
lxc config trust remove FINGERPRINT
lxc config trust remove REMOTE:FINGERPRINT
```

If just using `resource.name`, the 2nd and 3rd will work, but the first will have `resource.name` be empty.

Reply:
@tomponline

Reply:
@tomponline they're equivalent, we just messed up this particular command when implementing it and should have only supported the 3rd one.

We can't have ParseServers handle that format because it's a mix of remote and non-remote, ParseServers expects remotes.

Reply:
Ah OK so we are supporting variant 1. for historical reasons and we cannot remove support for it?

If so, then can you add a comment explaining why this is here for future reference, then its good to go. Thanks